workflows/phageannotator/main.nf

```diff
@@ -166,6 +168,34 @@ workflow PHAGEANNOTATOR {
     }
 
 
+    /*----------------------------------------------------------------------------
+        Extend viral contigs
+    ------------------------------------------------------------------------------*/
+    // if run_cobra == true, run subworkflow
```

Comment:
Depends on your commenting philosphy, but I think line 175 is sufficiently clear of that :)

Reply:
Yeah this is redundant 😅 I add these comments before I start writing the code and haven't been removing them